examples/gnrc_border_router/README.md

```diff
@@ -60,3 +60,98 @@ for further help.
 [1] https://tools.ietf.org/html/rfc1055
 
 [2] https://github.com/contiki-os/contiki/blob/master/tools/tunslip.c
+
+
+# gnrc_border_router with ethos
+A new setup is possible thanks to \#4438(https://github.com/RIOT-OS/RIOT/pull/4438) on which only one serial interface is needed. This is done through the ethos (Ethernet Over Serial) driver that multiplexes serial data between ethernet packets and shell commands.
+
+## Requirements
+This functionality works only in Linux machines, Mac support will be added in the future (lack of `tap` interface), and Windows support should possible by setting up the needed `tap` interface with an external program (not tested).
+
+## Setup
+Take care about the modification of the `Makefile`, since by default SLIP is enabled.
+In order to select ethos as the driver, you should modify some lines as following:
+```make
+#CFLAGS += -DSLIP_UART=$(SLIP_UART)
+#CFLAGS += -DSLIP_BAUDRATE=$(SLIP_BAUDRATE)
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+```
+Then deactivate the slip module and activate ethos
+```make
+# Include SLIP package for IP over Serial communication
+#USEMODULE += gnrc_slip
+USEMODULE += ethos gnrc_netdev2
+```
+This will allow to compile the BR firmware using ethos.
+Thus:
+```bash
+$ make clean all flash
+```
+should compile the example and flash it to your board.
+
+Afterwards, you need to compile the ethos tool instead of tunslip.
+To do this, in `dist/tools/ethos` you'll find the tool, just compile it with:
+```bash
+$ make
+```
+
+## Usage
+Once you flashed the BR example on your board, you should intialize your IP settings by choosing a prefix for your 6lo network. Just type `make term` and setup your BR:
```

Comment:
Ah the tun isn't initialized yet... Got it. (but maybe write below to kill this `make term` to prevent multiplexing issues over multiple terminals)